features/raadpleeg-bewoning-met-periode/bewoning-vaste-volgorde-mogelijke-bewoner.feature

```diff
@@ -0,0 +1,169 @@
+# language: nl
+@api
+Functionaliteit: De sortering van mogelijke bewoners op vaste volgorde
+Een mogelijke bewoner is een persoon die (voor een deel) van de gevraagde periode in de onzekerheidsperiode van de gevraagde adreshouding valt
+
+  Achtergrond:
+    Gegeven adres 'A1' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      |                 0800 |                         0800010000000001 |
+    En adres 'A2' heeft de volgende gegevens
+      | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+      |                 0800 |                         0800010000000002 |
+
+  Regel: Binnen een bewoning samenstelling worden de mogelijke bewoners in een vaste volgorde gesorteerd.
+   De sortering gebeurt op basis van de volgende criteria, in deze volgorde:
+    1. Datum van aanvang adreshouding: bewoners worden gesorteerd op de datum waarop zij op het adres zijn ingeschreven.
```

Comment:
`bewoners worden gesorteerd` veranderen in `bewoners worden oplopend gesorteerd`

Reply:
misschien is de volgende regel duidelijker: (mogelijke) bewoners worden op basis van datum aanvang adreshouding oplopend gesorteerd zodat de eerste (mogelijke) bewoner op het adres als eerste in de (mogelijke) bewoners lijst staat